index.src.html

```diff
@@ -332,6 +332,32 @@ <h3 dfn export id="referrer-policy-origin" oldids="referrer-policy-state-origin"
     priori</em> authenticated URLs</a>.
   </div>
 
+  <h3 dfn export id="referrer-policy-safe-origin" oldids="referrer-policy-state-safe-origin">"<code>safe-origin</code></h3>
```

Comment:
This is missing a `"` (double quote) between `</code>` and `</h3>`.

Without that the links to that section don't work and bikeshed issues an error:

    LINK ERROR: No 'dfn' refs found for '"safe-origin"'.